### ISSUE TITLE
batch norm layer

### ISSUE THREAD

**Issue by WHAAAT** [affinelayer/pix2pix-tensorflow#68]
Hello,
I've a question: 
It seems for your batchnorm layer, that you dont distinguish between training and testing phase. 
Is this correct? 

Thanks in advance!

**Comment by WHAAAT** [comment]
Oh I see, the paper describes this problem. 
They use the same settings for testing and training.

However restoring my model leads to worse results. any ideas about that?

**Comment by nikhilchh** [comment]
I also face the same issue.
Batch norm layer creates issue while testing. 
It gives weird results.

**Comment by vythaihn** [comment]
Any further thoughts about this? Do we have a choice to not include batch norm for test mode?
